tests/api/storage/sampletest_file_download.json

```diff
@@ -0,0 +1 @@
+/home/basanta/LF/Project/datamegh/tests/api/storage/sampletest_file.json
```

Comment:
I'm not clear what are you trying to do here? First it's not a valid json file and second it is specific to your environment.

Reply:
Things specific to ones environment needs to be gitignored.